File: src/main/java/com/hsmchurch/app/noticeboard/ui/request/NoticeDeleteRequest.java
package com.hsmchurch.app.noticeboard.ui.request;

import com.fasterxml.jackson.databind.PropertyNamingStrategy;
import com.fasterxml.jackson.databind.annotation.JsonNaming;
import lombok.Data;

@Data
@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
public class NoticeDeleteRequest {

    private Long boardId;
    private Long writerId;

    public NoticeDeleteRequest(final Long boardId,
                               final Long writerId) {
        this.boardId = boardId;
        this.writerId = writerId;
    }
}

File: src/main/java/com/hsmchurch/app/noticeboard/ui/request/NoticeDeleteRequest.kt
package com.hsmchurch.app.noticeboard.ui.request

import com.fasterxml.jackson.databind.PropertyNamingStrategy
import com.fasterxml.jackson.databind.annotation.JsonNaming

@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy::class)
data class NoticeDeleteRequest(val boardId: Long,
                               val writerId: Long)
